elide-graphql/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -121,11 +122,22 @@ private void filterSortPaginateSanityCheck(Environment environment) {
      * @param environment Environment encapsulating graphQL's request environment
      */
     private void logContext(RelationshipOp operation, Environment environment) {
-        List<Field> children = (environment.field.getSelectionSet() != null)
+        List<?> children = (environment.field.getSelectionSet() != null)
                 ? (List) environment.field.getSelectionSet().getChildren()
                 : new ArrayList<>();
-        String requestedFields = environment.field.getName() + (children.size() > 0
-                ? "(" + children.stream().map(Field::getName).collect(Collectors.toList()) + ")" : "");
+
+        List<String> fieldName = new ArrayList<String>();
+        if (children.size() > 0) {
+            children.stream().forEach(i -> { if (i.getClass().equals(Field.class)) {
+                    fieldName.add(((Field) i).getName());
+                } else {
+                    fieldName.add(((FragmentSpread) i).getName());
```

Comment:
This might not be safe (there are other types in graphQL besides fields and fragments).  Let's not bother logging the fragment spreads.